content/wiki/contests/NWERC2022.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "NWERC 2022"
+date: 2022-11-29T21:57:18+01:00
+menu: "wiki"
+---
+
+|            |                                 |
+|------------|--------------------------------:|
+| Host       |  Delft University of Technology |
+| Organizer  | W.I.S.V. \`Christiaan Huygens\` |
+| Commitee   |                [Chipcie][email] |
+| NWERC Date |             November 25-27 2022 | 
+| website    |                 [link][website] |
+| Pictures   |   (unavailable) [link][photos]  |
+| Scoreboard |              [link][scoreboard] |
+| Teams      |                             140 |
+
+## Problemsets
+
+### Practice
+
+| Contest          | Problemset, solutions and presentation                                            |
+|------------------|-----------------------------------------------------------------------------------|
+| Problemset       | [link][practiceset]                                                               |
+| Solutions slides | [link](https://2022.nwerc.eu/main/solutions.pdf)                                  |
+| Problem packages | [link](https://2022.nwerc.eu/test-session/solutions.zip) |
```

Comment:
```suggestion
| Contest          | Problemset, solutions and presentation                     |
|------------------|------------------------------------------------------------|
| Problemset       | [link](https://2022.nwerc.eu/test-session/problem-set.pdf) |
| Solutions slides | [link](https://2022.nwerc.eu/test-session/solutions.pdf)   |
| Problem packages | [link](https://2022.nwerc.eu/test-session/solutions.zip)   |
```

Reply:
These should be fixed by the latest commit.